diff --git a/build.gradle b/build.gradle
index 6bf1fd9c..942e4bcf 100644
--- a/build.gradle
+++ b/build.gradle
@@ -4,6 +4,7 @@ buildscript {
     repositories {
         google()
         jcenter()
+        mavenLocal()
     }
     dependencies {
         classpath Deps.tools_androidgradle
@@ -34,6 +35,7 @@ allprojects {
             url "https://repo.leanplum.com/"
         }
         jcenter()
+        mavenLocal()
     }
     tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
         kotlinOptions.jvmTarget = "1.8"
