diff --git a/build.gradle b/build.gradle
index a33fe20d1..558bd8d6c 100644
--- a/build.gradle
+++ b/build.gradle
@@ -8,6 +8,7 @@ buildscript {
     repositories {
         google()
         jcenter()
+        mavenLocal()
     }
 
     dependencies {
@@ -26,7 +27,7 @@ allprojects {
     repositories {
         google()
         jcenter()
-
+        mavenLocal()
         maven {
             url "https://maven.mozilla.org/maven2"
         }
