diff --git a/android/build.gradle b/android/build.gradle
index 4b38f16..5499f46 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -7,7 +7,7 @@ buildscript {
         mavenLocal()
         mavenCentral()
         google()
-        jcenter()
+        maven { url System.getenv("GRADLE_MAVEN_REPO") }
     }
     dependencies {
         classpath "com.android.tools.build:gradle:${androidplugin}"
@@ -82,6 +82,7 @@ repositories {
     mavenLocal()
     mavenCentral()
     google()
+    maven { url System.getenv("GRADLE_MAVEN_REPO") }
     maven { url "https://raw.githubusercontent.com/guardianproject/gpmaven/master" }
 }
 
diff --git a/android_tor_installer/build.gradle b/android_tor_installer/build.gradle
index d53793f..5e70c03 100644
--- a/android_tor_installer/build.gradle
+++ b/android_tor_installer/build.gradle
@@ -7,6 +7,7 @@ buildscript {
         mavenLocal()
         mavenCentral()
         google()
+        maven { url System.getenv("GRADLE_MAVEN_REPO") }
     }
     dependencies {
         classpath "com.android.tools.build:gradle:${androidplugin}"
@@ -48,7 +49,7 @@ repositories {
     mavenLocal()
     mavenCentral()
     google()
-    jcenter()
+    maven { url System.getenv("GRADLE_MAVEN_REPO") }
     maven { url "https://raw.githubusercontent.com/guardianproject/gpmaven/master" }
 
 }
diff --git a/build.gradle b/build.gradle
index d432247..d549cc2 100644
--- a/build.gradle
+++ b/build.gradle
@@ -9,6 +9,7 @@ subprojects {
     mavenLocal()
     mavenCentral()
     maven { url "http://repo.spring.io/plugins-release" }
+    maven { url System.getenv("GRADLE_MAVEN_REPO") }
   }
 }
 
